ee/tabby-ui/app/layout.tsx

```diff
@@ -1,5 +1,6 @@
 import { Metadata } from 'next'
-import { Toaster } from 'react-hot-toast'
+
+import { Toaster } from '@/components/ui/toaster'
```

Comment:
https://ui.shadcn.com/docs/components/sonner
https://ui.shadcn.com/docs/components/toast

Which one would you prefer? (in case you missed sonner)

Reply:
Yes, I missed sonner, sonner is better!